client/src/components/Header/SearchBar/LocationSearch/LocationSearch.scss

```diff
@@ -0,0 +1,39 @@
+@import "../../../../theme";
+
+.autocomplete-container {
+    display: inline-flex;
+    position: relative;
+
+    &, input {
+        height: 100%;
+    }
+}
+
+.autocomplete-dropdown-container {
+    position: absolute;
+    top: 30px;
+    background-color: #ffffff;
+    border: 1px solid rgba(0,0,0,0.2);
+    // This has to be absurdly large due to the leaflet map's absurdly large z-index
+    z-index: 1000;
```

Comment:
Nice

Reply:
I could lower the map's z-index but there might be a reason it's that high 🤷‍♂